data/com.redhat.Yggdrasil1.Worker1.foreman.service.in

```diff
@@ -0,0 +1,13 @@
+[Unit]
+Description=rhc-worker-playbook worker service
+Documentation=https://github.com/theforeman/foreman_ygg_worker
+
+[Service]
+Type=dbus
+User=root
+Group=@worker_user@
```

Comment:
It's confusing to me to use `worker_user` as a variable to determine the group. I also wonder what's the benefit of running as root but with a different group.

Reply:
This is a side-effect of the yggd D-Bus architecture. In order to own bus names
under the namespace "com.redhat.Yggdrasil1.Worker1", a process must have the
permission to do so. yggdrasil installs a D-Bus policy that restricts processes
that are members of a group (defined at compile-time).
https://github.com/RedHatInsights/yggdrasil/commit/16fec576da5b4d436abdc5c3cad7b2a33cca637

Reply:
Can we at least rename it to `@worker_group@` to emphasize what it's used for?

Reply:
Yea, good point.